app/src/main/res/layout/fragment_favor_accept_view.xml

```diff
@@ -90,15 +90,23 @@
                 android:textSize="15sp"
                 android:textColor="#FFFFFF"
                 android:layout_marginLeft="10dp" />
-
+            <!--
             <com.google.android.material.floatingactionbutton.FloatingActionButton
                 android:id="@+id/location_accept_view_btn"
                 android:layout_width="wrap_content"
 
                 android:layout_height="wrap_content"
                 android:layout_weight="0.1"
                 android:background="#FFFFFF"
-                android:src="@drawable/ic_location_on_24dp" />
+                android:drawableEnd="@drawable/ic_location_on_24dp" />
+            -->
+            <Button
```

Comment:
On my display the green looks almost indistinguishable from the purple background. Maybe we should make the button a bit larger?

Reply:
fixed